kuma/api/v1/views.py

```diff
@@ -387,6 +388,7 @@ def send_subscriptions_feedback(request):
 
 
 @api_view(["POST"])
+@login_required
```

Comment:
Aren't you glad you added these tests because you had almost forgotten about this. 
However, won't Django's `login_required` trigger a 302 to the login page?
That might make more sense in a traditional django view world. For an API, where you hit this view with a `fetch()` a redirect to a HTML page doesn't make sense. 

I actually think you should drop this decorator and weave it into the area where you check for the waffle flag. 
E.g. 
```python
def create_subscription(request):
    if not request.user.is_authenticated or not flag_is_active(request, "subscription"):
      ...403...
```

Reply:
Yup, I sure am. And good point wrt redirect. I changed it as recommended!